src/lib.ts

```diff
@@ -969,9 +969,5 @@ export function pickBjorn(mode: PickBjornMode = PickBjornMode.FREE): BjornedFami
 }
 
 export function kramcoGuaranteed(): boolean {
-  return (
-    have($item`Kramco Sausage-o-Matic™`) &&
-    totalTurnsPlayed() - get("_lastSausageMonsterTurn") + 1 >=
-      5 + 3 * get("_sausageFights") + Math.pow(Math.max(0, get("_sausageFights") - 5), 3)
-  );
+  return have($item`Kramco Sausage-o-Matic™`) && getKramcoWandererChance() === 1;
```

Comment:
```suggestion
  return have($item`Kramco Sausage-o-Matic™`) && getKramcoWandererChance() >= 1;
```

just in case - floating point errors etc